judo-runtime-core-dao-core/src/main/java/hu/blackbelt/judo/runtime/core/dao/core/processors/UpdatePayloadDaoProcessor.java

```diff
@@ -411,7 +412,7 @@ private void mergePayloads(final EReference mappedReference,
                                 parentInstanceGraph.getId(), false
                         ));
             } else if (updateIdentifier != null && originalIdentifier == null) {
-                checkState(!isContainment, "Identifier cannot be set on new association reference element: " +
+                checkState(!isContainment, "Identifier cannot be set on new composition reference element: " +
```

Comment:
Shouldn't this also be `isEmbedded`?

Reply:
No, implicit create is only enabled when the original payload doesn't contain any references. (In this line, the original error message was incorrect)

Reply:
No, because if the aggregation is an association in Entity level you can add it as reference or an attachment. (Here the error message was incorrect.)